src/components/ExistingProofDialog.vue

```diff
@@ -0,0 +1,102 @@
+<template>
+  <v-dialog>
+    <v-card>
+      <v-card-title>
+        {{ $t('UserDashboard.LatestProofs') }} <v-btn style="float:right;" variant="text" density="compact" icon="mdi-close" @click="close"></v-btn>
+      </v-card-title>
+      <v-divider></v-divider>
+      <v-card-text>
+        <v-row>
+          <v-col cols="6" sm="6" md="3" v-for="proof in userProofList" :key="proof">
+            <ProofCard :proof="proof" :showProofFooter="false" :isSelectable="true" :isSelected="selectedProof === proof" @proofSelected="selectProof"></ProofCard>
```

Comment:
i was thinking of doing a horizontal version of the ProofCard, but can be done after this PR & #375

Reply:
but like toelemon mentions, maybe showing part or all of the footer could still be useful, to have context on each proof